jdaviz/core/loaders/parsers/regions.py

```diff
@@ -0,0 +1,33 @@
+from functools import cached_property
+from regions import Regions
+from specutils import SpectralRegion
+
+from jdaviz.core.loaders.parsers import BaseParser
+from jdaviz.core.registries import loader_parser_registry
+
+
+@loader_parser_registry('Regions')
+class RegionsParser(BaseParser):
+    @property
+    def is_valid(self):
+        if self.app.config != 'specviz':
+            # NOTE: temporary during deconfig process
+            return False
+        if isinstance(self.input, (Regions, SpectralRegion)):
+            return True
+        if isinstance(self.input, str):
+            ext = self.input.split('.')[-1]
+            # once deconfigging is complete:
+            # return ext in ('reg', 'fits', 'ecsv')
+            return ext == 'ecsv'
```

Comment:
Is just the extension check enough? Besides "reg", the rest are pretty generic formats and are not confined to just storing regions. Given the config check would be removed in the future, what if the ECSV is actually a catalogs table, and so on?